.github/ISSUE_TEMPLATE/prepare-release.md

```diff
@@ -10,45 +10,69 @@ assignees: sergei-maertens
-  - [ ] Accounts (`openforms.accounts.tests.test_oidc`)
-  - Appointments: Qmatic (`openforms.appointments.contrib.qmatic`) (no testenv available anymore)
-  - Authentication plugins
-    - [ ] DigiD (Signicat): `openforms.authentication.contrib.digid.tests.test_signicat_integration`
-    - [ ] eHerkenning
-          (Signicat):`openforms.authentication.contrib.eherkenning.tests.test_signicat_integration`
-    - [ ] DigiD/EH via OIDC: `openforms.authentication.contrib.digid_eherkenning_oidc`
-    - [ ] Org via OIDC: `openforms.authentication.contrib.org_oidc`
-    - [ ] `openforms.tests.test_registrator_prefill`
+
+  - Accounts
+
+    - [ ] `openforms.accounts.tests.test_oidc`
+
+  - Authentication:
+
+    - [ ] `openforms.authentication.contrib.digid.tests.test_signicat_integration`
+    - [ ] `openforms.authentication.contrib.digid_eherkenning_oidc.tests.test_auth_context_data`
+    - [ ] `openforms.authentication.contrib.digid_eherkenning_oidc.tests.test_auth_flow_callbacks`
+    - [ ] `openforms.authentication.contrib.digid_eherkenning_oidc.tests.test_auth_flow_init`
+    - [ ] `openforms.authentication.contrib.digid_eherkenning_oidc.tests.test_auth_flow_logout`
+    - [ ] `openforms.authentication.contrib.org_oidc.tests.test_auth_context_data`
+    - [ ] `openforms.authentication.contrib.org_oidc.tests.test_auth_flow_callbacks`
+    - [ ] `openforms.authentication.contrib.org_oidc.tests.test_auth_flow_init`
+    - [ ] `openforms.authentication.contrib.org_oidc.tests.test_auth_flow_logout`
+    - [ ] `openforms.authentication.contrib.eherkenning.tests.test_signicat_integration`
+
   - General purpose clients
+
     - [ ] `openforms.contrib.brk`
+    - [ ] `openforms.contrib.customer_interactions`
     - [ ] `openforms.contrib.haal_centraal.tests.test_integration`
     - [ ] `openforms.contrib.kvk`
-    - [ ] `openforms.contrib.objects_api.tests`
-    - `suwinet.tests.test_client` (testenv access has been retracted and won't be reinstated)
-  - Family members
-    - [ ] `openforms.prefill.contrib.family_members.tests.test_plugin`
-    - [ ] `openforms.emails.tests.test_digest_functions`
-  - Forms
-    - [ ] `openforms.forms.tests.test_import_export`
-    - [ ] `openforms.forms.tests.e2e_tests.test_registration_backend_conf`
-    - [ ] `openforms.formio.formatters.tests.test_default_formatters`
-  - Payment plugins
-    - [ ] Ogone legacy: `openforms.payments.contrib.ogone.tests.test_client`
-    - [ ] Worldline: `openforms.payments.contrib.worldline.tests.test_plugin`
-  - Prefill
-    - [ ] Objects API: `openforms.prefill.contrib.objects_api`
-    - Suwinet: `openforms.prefill.contrib.suwinet` (testenv access has been retracted and won't be
-      reinstated)
-  - ReferenceLists:
+    - [ ] `openforms.contrib.objects_api`
     - [ ] `openforms.contrib.reference_lists`
-    - [ ] `openforms.emails.tests.test_tasks_integration`
+    - [ ] `soap.tests.test_client.ClientTransportTests`
```

Comment:
What about this one?I see we haven't recorded anything since 2024. I think I have missed that in the previous release cycles.

Reply:
should be fine to re-record, if I run into issues I'll update the template!